neo/Consensus/ChangeView.cs

```diff
@@ -4,7 +4,10 @@ namespace Neo.Consensus
 {
     public class ChangeView : ConsensusMessage
     {
-        public byte NewViewNumber;
+        /// <summary>
+        /// NewViewNumber is always set to the current ViewNumber asking changeview + 1
+        /// </summary>
+        public byte NewViewNumber => ViewNumber++;
```

Comment:
```suggestion
        public byte NewViewNumber => ViewNumber + 1;
```

Reply:
I tried here, it says "Use block body for properties"

Reply:
`ViewNumber++` will change the value of `ViewNumber`.

Reply:
kkkkkkkkk, vishhh

Reply:
`++ViewNumber` ?

Reply:
"Consensus/ChangeView.cs(10,38): error CS0266: Cannot implicitly convert type 'int' to 'byte'. An explicit conversion exists (are you missing a cast?) [/opt/neoLib/neo/neo.csproj]"

Maybe we need that Get functions.

Reply:
public byte NewViewNumber => (byte)(ViewNumber + 1);